client/ayon_shotgrid/plugins/publish/integrate_shotgrid_publish.py

```diff
@@ -52,15 +53,51 @@ def process(self, instance):
                 query_filters
             )
 
-            if platform.system() == "Windows":
-                local_path = local_path.replace("\\", "/")
+            sg_local_store = sg.find_one(
```

Comment:
This throws `NameError`, it should be

```suggestion
            sg_local_store = sg_session.find_one(
```